database/factories/CommentFactory.php

```diff
@@ -8,12 +8,6 @@
 
 $factory->define(Comment::class, function (Faker $faker) {
     return [
-        'job_poster_id' => function () {
```

Comment:
job_poster_id and user_id should be un-deleted. They can be overridden in the seeder, but they need a default value.